squarelet/organizations/forms.py

```diff
@@ -259,3 +271,91 @@ def clean(self):
         if good_organization and good_organization == bad_organization:
             raise forms.ValidationError("Cannot merge an organization into itself")
         return cleaned_data
+
+
+class ProfileChangeRequestForm(forms.ModelForm):
+    """Request changes to core organization profile data"""
+
+    url = forms.URLField(label=_("URL"), required=False)
+
+    class Meta:
+        model = ProfileChangeRequest
+        fields = ["name", "slug", "url", "city", "state", "country", "explanation"]
+
+    def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop("request", None)
+        super().__init__(*args, **kwargs)
+
+        # Populate placeholders with current organization values
+        if self.instance and self.instance.organization:
+            org = self.instance.organization
+            self.fields["name"].widget.attrs["placeholder"] = org.name
+            self.fields["slug"].widget.attrs["placeholder"] = org.slug
+            self.fields["city"].widget.attrs["placeholder"] = org.city or _("City")
+            self.fields["state"].widget.attrs[
+                "placeholder"
+            ] = org.get_state_display() or _("State")
+            self.fields["country"].widget.attrs[
+                "placeholder"
+            ] = org.get_country_display() or _("Country")
+
+            # Update help text to show current values
+            self.fields["name"].help_text = _(
+                f"Current: {org.name}. Leave blank to keep unchanged."
+            )
+            self.fields["slug"].help_text = _(
+                f"Current: {org.slug}. Leave blank to keep unchanged."
+            )
+            self.fields["city"].help_text = _(
+                f"Current: {org.city or 'Not set'}. Leave blank to keep unchanged."
+            )
+            self.fields["state"].help_text = _(
+                f"Current: {org.get_state_display() or 'Not set'}. "
+                "Leave blank to keep unchanged."
+            )
+            self.fields["country"].help_text = _(
+                f"Current: {org.get_country_display() or 'Not set'}. "
+                "Leave blank to keep unchanged."
+            )
+            self.fields["url"].help_text = _(
+                "Add a URL to associate with this organization."
+            )
+            self.fields["explanation"].help_text = _(
+                "Explain why you are requesting these changes "
+                "(required for staff review)."
+            )
```

Comment:
As a user, it feels clearer to me to have the fields prefilled with their current value instead of with the placeholder + help text approach. We could then check for value changes when cleaning the field.